File: nelio-alves-algoritmo-e-logica-de-programacao/c/exercicios/mais_velho.c
#include <stdio.h>

int main() {
    int N, posicao, maiorIdade;

    printf("Quantas pessoas voce vai digitar? ");
    scanf("%d", &N);

    char nomes[N][50];
    int idades[N];

    for (int i = 0; i < N; i++) {
        printf("Dados da %da pessoa:\n", i + 1);
        printf("Nome: ");
        scanf("%s", &nomes[i][0]);
        printf("Idade: ");
        scanf("%d", &idades[i]);
    }

    posicao = 0;
    maiorIdade = idades[0];

    for (int i = 1; i < N; i++) {
        if (idades[i] > maiorIdade) {
            maiorIdade = idades[i];
            posicao = i;
        }
    }

    printf("PESSOA MAIS VELHA: %s\n", nomes[posicao]);

    return 0;
}

File: nelio-alves/algoritmo-e-logica-de-programacao/python/exercicios/mais_velho.py
N: int; posicao: int; maiorIdade: int

N = int(input("Quantas pessoas voce vai digitar? "))

nomes: [str] = [0 for x in range(N)]
idades: [int] = [0 for x in range(N)]

for i in range(N):
	print(f"Dados da {i+1}a pessoa:")
	nomes[i] = str(input("Nome: "))
	idades[i] = int(input("Idade: "))

posicao = 0
maiorIdade = idades[0]

for i in range(N):
	if idades[i] > maiorIdade:
		maiorIdade = idades[i]
		posicao = i

print(f"PESSOA MAIS VELHA: {nomes[posicao]}")